### ISSUE TITLE
plantuml files are not created

### ISSUE THREAD

**Issue by floeffler-dimabay** [phpDocumentor/phpDocumentor#3031]
## Expected behavior
docker run --rm -v $PWD:/d -w /d/api phpdoc/phpdoc run

-> docs a created and plantuml files are created

## Actual behavior
-> The docs are created but not the plantuml files.

```
15:12:19 WARNING   [php] Warning: file_put_contents(<?xml version="1.0" encoding="UTF-8" standalone="no"?><svg xmlns="http://...@startuml

skinparam shadowing false
skinparam linetype ortho
hide empty members
left to right direction
set namespaceSeparator \\


class \\Company\\Auth\\HashedPassword {
}
@enduml

PlantUML version 1.2021.10(Mon Aug 30 13:43:48 UTC 2021)
(GPL source distribution)
Java Runtime: OpenJDK Runtime Environment
JVM: OpenJDK 64-Bit Server VM
Default Encoding: ANSI_X3.4-1968
Language: en
Country: US
--></g></svg>): failed to open stream: Invalid argument
```

## Environment
Docker sktop 4.1.0 (on Mac)
Docker Container: php:7.4-apache
Docker Container: phpdoc/phpdoc

phpdoc.dist.xml
```
<?xml version="1.0" encoding="UTF-8" ?>
<phpdocumentor
        configVersion="3"
        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
        xmlns="https://www.phpdoc.org"
        xsi:noNamespaceSchemaLocation="https://docs.phpdoc.org/latest/phpdoc.xsd"
>
    <title>API</title>
    <paths>
        <output>./.build-phpdoc</output>
        <cache>./var/build-phpdoc/cache</cache>
    </paths>
    <version number="3.0.0">
        <folder>latest</folder>
        <api format="php">
            <default-package-name>COMPANY</default-package-name>
            <source dsn=".">
                <path>src</path>
            </source>
            <output>api</output>
            <extensions>
                <extension>php</extension>
            </extensions>
            <markers>
                <marker>TODO</marker>
                <marker>FIXME</marker>
            </markers>
            <include-source>true</include-source>
        </api>
        <guide format="rst">
            <source dsn=".">
                <path>src</path>
            </source>
            <output>guide</output>
        </guide>
    </version>
    <setting name="guides.enabled" value="true"/>
    <setting name="graphs.enabled" value="true"/>
    <template name="default"/>
</phpdocumentor>
```

**Comment by floeffler-dimabay** [comment]
Seems like the filename for the plantuml-file is missing?!

**Comment by mvriel** [comment]
Thanks for reporting this issue; I have just pushed a fix to master that should resolve the situation. Apparently we had mixed up the argument order of file_put_contents there (and yes, I am disappointed in myself now ;) ).

During testing I did find that the PlantUML diagrams need more work; it is not the clearest information out there right now and we only use the implements and extends information to create it, and not how it is used in methods and properties.

These fixes will come, but I cannot give a timeline at the moment as our focus is on guide generating
